analysis/codelists.py

```diff
@@ -0,0 +1,51 @@
+from databuilder.codes import REGISTRY, Codelist, codelist_from_csv
+
+
+def codelist(codes, system):
+    code_class = REGISTRY[system]
+    return Codelist(
+        codes={code_class(code) for code in codes},
+        category_maps={},
+    )
+
+
+covid_icd10 = codelist_from_csv(
+    "codelists/opensafely-covid-identification.csv",
```

Comment:
If you wanted to keep it simple but still use `pathlib` you could write this as:
```py
CODELIST_DIR = Path("codelists")
```
(I needed to use the `Path(__file__)` stuff in my original test study because the codelists weren't in the usual location, but it's a bit intimidating for someone who doesn't know Python.) 

I think there's something to be said for having the directory defined as a variable so if you do have to move or rename it then there's only one place to change. But in practice, I don't anyone will ever be moving this directory so maybe it's not worth it. Happy to leave it to your judgement as to what's most readable.

Reply:
done in bb1d3268991f25edc778b3a8a24f33043ab8c78d